<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parting_dialog</class>
 <widget class="QDialog" name="parting_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>663</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Parting Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="speed_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>surface speed
[units/min]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="feed_lineEdit">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="openPushButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>open in new tab</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="feed_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>initial feedrate
[units/min]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="11">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="diameter_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>nominal diameter
[units]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="spi_lineEdit">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="diameter_label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>segments 
per inch</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="diameter_lineEdit">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="unit_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>units</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="generatePushButton">
     <property name="text">
      <string>generate gcode</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="unit_comboBox">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="speed_lineEdit">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="11" column="0" rowspan="2" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>feed_lineEdit</tabstop>
  <tabstop>speed_lineEdit</tabstop>
  <tabstop>diameter_lineEdit</tabstop>
  <tabstop>spi_lineEdit</tabstop>
  <tabstop>unit_comboBox</tabstop>
  <tabstop>generatePushButton</tabstop>
  <tabstop>openPushButton</tabstop>
  <tabstop>plainTextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
